dist/angular-validation.js

```diff
@@ -382,6 +382,7 @@
         messageElem.css('display', '');
       } else {
         messageElem.css('display', 'none');
+        messageElem.html('');
```

Comment:
For the initial design, I'm not going to delete the message span. It will break other directive like  `no-validation-message`